restapi/src/test/java/io/stargate/web/docsapi/service/util/DocsApiUtilsTest.java

```diff
@@ -108,6 +108,40 @@ public void numberFormatException() {
     }
   }
 
+  @Nested
+  class ConvertUnicodeCodePoints {
+
+    @Test
+    public void happyPath() {
+      String result = DocsApiUtils.convertUnicodeCodePoints("\\u002E is a period");
+
+      assertThat(result).isEqualTo(". is a period");
+
+      result =
+          DocsApiUtils.convertUnicodeCodePoints("I can represent braces too: \\u005b000\\u005d");
+      assertThat(result).isEqualTo("I can represent braces too: [000]");
+
+      result =
+          DocsApiUtils.convertUnicodeCodePoints(
+              "\\u but without valid code points after are ignored: \\ufg00");
+      assertThat(result).isEqualTo("\\u but without valid code points after are ignored: \\ufg00");
+    }
+
+    @Test
+    public void arraySegmentsIgnored() {
+      String result = DocsApiUtils.convertUnicodeCodePoints("[1],[44],[555]");
```

Comment:
Since there are no Unicode escapes here, why should the result be anything else than the original String?
(I am assuming encoding is handled elsewhere)

Reply:
This was just a blind copy paste from the corresponding test above - and I think I'm going to remove it :)